<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advDialog</class>
 <widget class="QDialog" name="advDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>490</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="SOCBox_2">
      <property name="title">
       <string>Variable standardization</string>
      </property>
      <widget class="QComboBox" name="varSTDComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>On</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="SOCBox">
      <property name="title">
       <string>Measure of Score of Change (SOC)</string>
      </property>
      <widget class="QComboBox" name="SOCComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>SOC-f</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SOC-RSS</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Initialization</string>
      </property>
      <widget class="QComboBox" name="initComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>GWR estimates</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OLS estimates</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Convergence threshold</string>
      </property>
      <widget class="QComboBox" name="convComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1e-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1e-3</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Monte Carlo test for spatial variability</string>
      </property>
      <widget class="QComboBox" name="mcComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On (Very Slow)</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Local collinearity diagnostics</string>
      </property>
      <widget class="QComboBox" name="locollinearComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Bandwidths confidence intervals</string>
      </property>
      <widget class="QComboBox" name="bwciComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="applyBTN">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBTN">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
